database/Seeders/eggs/source-engine/egg-ark--survival-evolved.json

```diff
@@ -40,46 +50,28 @@
             "name": "Admin Password",
             "description": "If specified, players must provide this password (via the in-game console) to gain access to administrator commands on the server.",
             "env_variable": "ARK_ADMIN_PASSWORD",
-            "default_value": "PleaseChangeMe",
+            "default_value": "",
             "user_viewable": true,
             "user_editable": true,
             "rules": "nullable|alpha_dash|between:1,100"
         },
         {
-            "name": "Server Map",
-            "description": "Available Maps: TheIsland, TheCenter, Ragnarok, ScorchedEarth_P, Aberration_P, Extinction, Valguero_P, Genesis, CrystalIsles",
-            "env_variable": "SERVER_MAP",
-            "default_value": "TheIsland",
-            "user_viewable": true,
-            "user_editable": true,
-            "rules": "required|string|max:20"
-        },
-        {
-            "name": "App ID",
-            "description": "ARK steam app id for auto updates. Leave blank to avoid auto update.",
-            "env_variable": "SRCDS_APPID",
-            "default_value": "376030",
-            "user_viewable": true,
-            "user_editable": false,
-            "rules": "nullable|numeric"
-        },
-        {
-            "name": "Server Name",
-            "description": "ARK server name",
-            "env_variable": "SESSION_NAME",
-            "default_value": "A Pterodactyl Hosted ARK Server",
+            "name": "Server Port",
+            "description": "ARK server port used by client.",
+            "env_variable": "PORT",
+            "default_value": "7777",
             "user_viewable": true,
             "user_editable": true,
-            "rules": "required|string|max:128"
+            "rules": "required|numeric"
         },
         {
             "name": "Use Rcon",
-            "description": "Enable or disable rcon system. (true or false)\r\n\r\nDefault True for the console to work.",
+            "description": "Enable or disable rcon system. (true or false)",
             "env_variable": "ENABLE_RCON",
-            "default_value": "True",
+            "default_value": "false",
```

Comment:
RCON is required and disabling it by default will flood Discord channels complaining why rcon fails to connect